### PR TITLE
feat(adapter): AWS Lambda Adapter supports requests via Lambda Function URLs.

### PR DESCRIPTION
**Pull request by watany-dev**
Lambda Function URLs are endpoints that allow for simple requests to be passed to Lambda without an API Gateway. 

https://aws.amazon.com/jp/blogs/aws/announcing-aws-lambda-function-urls-built-in-https-endpoints-for-single-function-microservices/

Most of the event is compatible with API Gateway, but the location of the method is different, which is why, as mentioned in https://github.com/honojs/hono/issues/1075, we specified the behavior of GET during a POST request. This patch attempts to fix that issue.

### CODE REVIEW

**Comment by yusukebe** [comment]
Hi @watany-dev !

Thank you for the PR. I'd like to try running the application with this PR in my environment. So, please wait until I've done it.

File: src/adapter/aws-lambda/handler.ts [hunk 1]
```diff
@@ -83,7 +98,7 @@ const createRequest = (event: APIGatewayProxyEvent | APIGatewayProxyEventV2) =>
 
   const requestInit: RequestInit = {
     headers,
-    method: event.httpMethod,
+    method: event.httpMethod ?? event.requestContext.http?.httpMethod,
```
Comment (yusukebe) [inline src/adapter/aws-lambda/handler.ts:101]: I tried it, but `event.requestContext.http?.httpMethod` is `undefined`. So, `method` must be:

```ts
const method = 'httpMethod' in event ? event.httpMethod : event.requestContext.http.method
```

**Review by yusukebe — CHANGES_REQUESTED** [review]
Hi @watany-dev !

Thank you for the PR.

As you pointed out, POST is not working on Lambda Function URLs, so this change seems to fix it. I've left a comment; please check it!

**Comment by watany-dev** [comment]
I have reviewed the workflow and understood the need to perform runtime tests. I have made the necessary modifications.

**Comment by watany-dev** [comment]
Sorry, I fixed the Format.

**Review by yusukebe — APPROVED** [review]

**Comment by yusukebe** [comment]
Thanks! I'll merge it now!

**Pull request merged**